cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -0,0 +1,174 @@
+{# ==========================================================================
+
+   filterable_list_controls.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create an expandable of filters when given:
+
+   settings:            Array of objects describing a form structure.
+                        Example nested form:
+                        {
+                            'type': 'form',
+                            'children': [
+                                {
+                                    'type': 'text',
```

Comment:
Left align the values here and throughout to stay consistent w/ the other macros.